package protoFile.item.UnloadXiangQian2113;
message UnLoadXiangQianRequest{
      required int32 id=1;//角色id
      required int32 equipId=2;//装备id
      required int32 type=3;//1背包2身上装备
      required int32 position = 4;//摘除宝石的位置
}
message UnLoadXiangQianResponse{
      optional string message=1;
      optional bool result=2;
}